.github/workflows/release-pr.yml

```
@@ -1,31 +1,31 @@
-# Create a PR for a release when a commit is pushed on a release/v* branch
-
+##
+# Create a PR for a release when a commit is pushed on a release/*-v* branch to support the releases of both the
+# operator and cluster charts
 name: release-pr
 
 on:
   push:
     branches:
-    - release/*-v*
+      - release/*
+
+permissions:
+  pull-requests: write
 
 jobs:
-  pull-request:
-    runs-on: ubuntu-22.04
+  create-pull-request:
+    runs-on: ubuntu-latest
```

As suggested by @sxd, can we keep it explicit?
```suggestion
    runs-on: ubuntu-22.04
```

---

I'll explain, the reason is that if for some reason there's a change in the latest version of the ubuntu image with different versions actions or commands could start to fail without any clear reason and finding that it was the image it will be a big pain in the ass

---

Currently `ubuntu-latest` is an alias for `ubuntu-22.04`. But with the upcoming 24.04 LTS in April it makes more sense to fix it to `latest`. This way we will benefit from the best support options for the newest release.